packages/ai/mcp-stdio/create-child-process.ts

```diff
@@ -33,7 +33,7 @@ function getDefaultEnvironment(): Record<string, string> {
         ]
       : ['HOME', 'LOGNAME', 'PATH', 'SHELL', 'TERM', 'USER'];
 
-  const env: Record<string, string> = {};
+  const env: Record<string, string> = customEnv ?? {};
 
```

Comment:
how can we test this? e.g. add `export` here and have a separate test file for this part?